Development/nmos/constraints.cpp

```diff
@@ -0,0 +1,165 @@
+
+#include "nmos/constraints.h"
+
+#include <map>
+#include <boost/algorithm/string/predicate.hpp>
+#include "nmos/capabilities.h"
+#include "nmos/json_fields.h"
+#include "nmos/sdp_utils.h"
+#include "nmos/video_jxsv.h"
```

Comment:
I hope that including _nmos/video_jxsv.h_ is not required here?

```suggestion
```

Reply:
Added `video_jxsv_parameter_constraints` in `nmos/video_jxsv.h` to decouple it.

Reply:
Replaced `video_jxsv_parameter_constraints` with `video_jxsv_flow_parameter_constraints` and `video_jxsv_sender_parameter_constraints`.